layers/+readers/elfeed/README.org

```diff
@@ -109,7 +109,7 @@ In this case you can disable these by setting =elfeed-enable-goodies= to =nil=.
 
 | Key binding | Description  |
 |-------------+--------------|
-| ~SPC a f~   | start elfeed |
+| ~SPC a r f~ | start elfeed |
```

Comment:
I think this should be `SPC a r e` instead

Reply:
Yes, good catch.  Fixed in latest push.